### PR TITLE
pages and workers changelog entries for github app perm update

### PR DESCRIPTION
**Pull request by aninibread**
### Summary

Updated in Pages and Workers changelog

### Documentation checklist

- [ ] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.
- [ ] If a larger change - such as adding a new page- an issue has been opened in relation to any incorrect or out of date information that this PR fixes.
- [ ] Files which have changed name or location have been allocated [redirects](https://developers.cloudflare.com/pages/configuration/redirects/#per-file).

### CODE REVIEW

File: src/content/changelogs/pages.yaml [hunk 1]
```diff
@@ -5,6 +5,15 @@ productLink: "/pages/"
 productArea: Developer platform
 productAreaLink: /workers/platform/changelog/platform/
 entries:
+  - publish_date: "2024-12-19"
+    title: Cloudflare GitHub App Permissions Update
+    description: |-
+      - Users with the [Cloudflare GitHub App](https://github.com/apps/cloudflare-workers-and-pages) installed to connect repositories to Pages or Workers will be prompted to update permissions **today**. New permissions will allow Cloudflare to automatically create a repository on your GitHub account and deploy the new repository for you.
```
Comment (WalshyDev) [inline src/content/changelogs/pages.yaml]: > will allow Cloudflare to automatically create a repository on your GitHub account and deploy the new repository for you.

Why? (i know why but the user reading doesn't)

File: src/content/changelogs/pages.yaml [hunk 2]
```diff
@@ -5,6 +5,15 @@ productLink: "/pages/"
...
+    description: |-
+      - Users with the [Cloudflare GitHub App](https://github.com/apps/cloudflare-workers-and-pages) installed to connect repositories to Pages or Workers will be prompted to update permissions **today**. New permissions will allow Cloudflare to automatically create a repository on your GitHub account and deploy the new repository for you. 
+      - **Requested permissions:** 
+        - [Repository Administration](https://docs.github.com/en/rest/authentication/permissions-required-for-github-apps?apiVersion=2022-11-28#repository-permissions-for-administration) (read/write) to create repositories
+        - [Contents](https://docs.github.com/en/rest/authentication/permissions-required-for-github-apps?apiVersion=2022-11-28#repository-permissions-for-contents) (read/write) to push code to the created repositories 
+      - **Action:** When you receive the email from GitHub with the subject “[GitHub] Cloudflare Workers & Pages is requesting updated permission”, please review and accept the new permissions. *If you take no action or choose to decline, you can continue to connect repositories and deploy changes via the Cloudflare GitHub App as you do today.*
```
Comment (WalshyDev) [inline src/content/changelogs/pages.yaml]: pet peeve of mine these macos quotes
```suggestion
      - **Action:** When you receive the email from GitHub with the subject "[GitHub] Cloudflare Workers & Pages is requesting updated permission", please review and accept the new permissions. *If you take no action or choose to decline, you can continue to connect repositories and deploy changes via the Cloudflare GitHub App as you do today.* 
```

File: src/content/changelogs/workers.yaml [hunk 3]
```diff
@@ -5,6 +5,15 @@ productLink: "/workers/"
...
+        - Users with the [Cloudflare GitHub App](https://github.com/apps/cloudflare-workers-and-pages) installed to connect repositories to Pages or Workers will be prompted to update permissions **today**. New permissions will allow Cloudflare to automatically create a repository on your GitHub account and deploy the new repository for you. 
+        - **Requested permissions:** 
+          - [Repository Administration](https://docs.github.com/en/rest/authentication/permissions-required-for-github-apps?apiVersion=2022-11-28#repository-permissions-for-administration) (read/write) to create repositories
+          - [Contents](https://docs.github.com/en/rest/authentication/permissions-required-for-github-apps?apiVersion=2022-11-28#repository-permissions-for-contents) (read/write) to push code to the created repositories 
+        - **Action:** When you receive the email from GitHub with the subject “[GitHub] Cloudflare Workers & Pages is requesting updated permission”, please review and accept the new permissions. *If you take no action or choose to decline, you can continue to connect repositories and deploy changes via the Cloudflare GitHub App as you do today.*
```
Comment (WalshyDev) [inline src/content/changelogs/workers.yaml]:
```suggestion
        - **Action:** When you receive the email from GitHub with the subject "[GitHub] Cloudflare Workers & Pages is requesting updated permission", please review and accept the new permissions. *If you take no action or choose to decline, you can continue to connect repositories and deploy changes via the Cloudflare GitHub App as you do today.* 
```

File: src/content/changelogs/pages.yaml (hunk 1 above)
Comment (irvinebroque) [inline src/content/changelogs/pages.yaml]:
```suggestion
      - Updated permissions that the [Cloudflare GitHub App](https://github.com/apps/cloudflare-workers-and-pages) requests when installed, allowing you to create new repositories on your GitHub account from the Cloudflare dashboard, when creating new projects. 
       If you have the [Cloudflare GitHub App](https://github.com/apps/cloudflare-workers-and-pages) installed, you will receive an email prompting you to update the permissions you grant the Cloudflare Github App.
```

- 2nd person, not "users"
- "today" is confusing if you imagine reading this changelog entry next week
- Invert it and start with the what and why, then the what will happen
- Can we clarify how this impacts new people?

**Comment by aninibread** [comment]
requesting review again: https://anni-gh-app-perm-update.cloudflare-docs-7ou.pages.dev/workers/platform/changelog/

File: src/content/changelogs/pages.yaml [hunk 4]
```diff
@@ -5,6 +5,18 @@ productLink: "/pages/"
 productArea: Developer platform
 productAreaLink: /workers/platform/changelog/platform/
 entries:
+  - publish_date: "2024-12-19"
+    title: Cloudflare GitHub App Permissions Update
+    description: |-
+      - Cloudflare is requesting updated permissions for the [Cloudflare GitHub App](https://github.com/apps/cloudflare-workers-and-pages) to enable features like automatically creating a repository on your GitHub account and deploying the new repository for you. This feature is coming out soon to support a better onboarding experience.
```
Comment (nevikashah) [inline src/content/changelogs/pages.yaml]: to enable features like automatically creating a repository on your GitHub account and deploying the new repository for you when getting started with a template.

File: src/content/changelogs/pages.yaml [hunk 5]
```diff
@@ -5,6 +5,18 @@ productLink: "/pages/"
...
+          - [Contents](https://docs.github.com/en/rest/authentication/permissions-required-for-github-apps?apiVersion=2022-11-28#repository-permissions-for-contents) (read/write) to push code to the created repositories.
+        - **Who is impacted:**
+          - Existing users will be prompted to update permissions when GitHub sends an email with subject "[GitHub] Cloudflare Workers & Pages is requesting updated permission" on December 19th, 2024.
+          - New users installing the app will see the updated permissions during the connecting repository process.
+        - **Action:** Review and accept the permissions update to use upcoming features. *If you decline or take no action, you can continue connecting repositories and deploying changes via the Cloudflare GitHub App as you do today, but new features requiring these permissions will not be available.*
+        - **Questions?** Visit [#github-permissions-update](https://discord.com/channels/595317990191398933/1313895851520688163) in Cloudflare Developers Discord.
```
Comment (WalshyDev) [inline src/content/changelogs/pages.yaml]:
```suggestion
        - **Questions?** Visit [#github-permissions-update](https://discord.com/channels/595317990191398933/1313895851520688163) in the Cloudflare Developers Discord.
```

File: src/content/changelogs/workers.yaml [hunk 6]
```diff
@@ -5,6 +5,18 @@ productLink: "/workers/"
...
+          - [Contents](https://docs.github.com/en/rest/authentication/permissions-required-for-github-apps?apiVersion=2022-11-28#repository-permissions-for-contents) (read/write) to push code to the created repositories.
+        - **Who is impacted:**
+          - Existing users will be prompted to update permissions when GitHub sends an email with subject "[GitHub] Cloudflare Workers & Pages is requesting updated permission" on December 19th, 2024.
+          - New users installing the app will see the updated permissions during the connecting repository process.
+        - **Action:** Review and accept the permissions update to use upcoming features. *If you decline or take no action, you can continue connecting repositories and deploying changes via the Cloudflare GitHub App as you do today, but new features requiring these permissions will not be available.*
+        - **Questions?** Visit [#github-permissions-update](https://discord.com/channels/595317990191398933/1313895851520688163) in Cloudflare Developers Discord.
```
Comment (WalshyDev) [inline src/content/changelogs/workers.yaml]:
```suggestion
        - **Questions?** Visit [#github-permissions-update](https://discord.com/channels/595317990191398933/1313895851520688163) in the Cloudflare Developers Discord.
```

**Review by WalshyDev — APPROVED** [review]

**Pull request merged**